### PR TITLE
[FSR] 72105 - Adding end of form Medallia survey button

### PR DESCRIPTION
**Pull request by Dr-Pongo**
## Summary
Adding button to trigger Medallia survey modal on the confirmation page of the FSR since popups are more intrusive and there were issues triggering it mid-form.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#72105

## Testing done
Manual testing complete. Will test in review instance as well

### Confirmation page alert
| Full FSR | Streamlined | 
| -------- | -------- |
| image | image |

## What areas of the site does it impact?
FSR

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by OptionSelect — DISMISSED** [review]
LGTM! Nice catch on the va-alerts as well with the prop

**Review by amponce — DISMISSED** [review]
Solid

File: src/applications/financial-status-report/components/shared/SurveyInformation.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <VaButton
+          onClick={window.KAMPYLE_ONSITE_SDK?.showForm(surveyNumber)}
+          text="Provide feedback"
+          uswds
+        />
+      </div>
+    );
+  }
+
+  Sentry.withScope(scope => {
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/financial-status-report/components/shared/SurveyInformation.jsx:41]: Sentry message just contains relevant info for troubleshooting

File: src/applications/financial-status-report/components/shared/SurveyInformation.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const message = '5655 Error loading end of form survey';
+    scope.setContext(message, {
+      KAMPYLE_ONSITE_SDK: window.KAMPYLE_ONSITE_SDK,
+      showForm: window.KAMPYLE_ONSITE_SDK?.showForm(41),
+    });
+    Sentry.captureMessage(message);
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/financial-status-report/components/shared/SurveyInformation.jsx:47]: Sentry message just contains relevant info for troubleshooting

**Review by OptionSelect — DISMISSED** [review]
LGTM! 🚀

**Review by amponce — APPROVED** [review]
👍 Looks good!

**Pull request merged**